services/web/server/src/simcore_service_webserver/api_keys/_controller/rest.py

```diff
@@ -55,8 +55,8 @@ async def create_api_key(request: web.Request):
     api_key = ApiKeyCreateResponse.model_validate(
         {
             **asdict(created_api_key),
-            "api_base_url": "http://localhost:8000",
-        }  # TODO: https://github.com/ITISFoundation/osparc-simcore/issues/6340 # @pcrespov
+            "api_base_url": get_api_base_url(request) or "",
```

Comment:
is empty really going to validate here? I prefer to have `URL | None` for this entry. Emtpy strings for `URL` have proven to be really misleading in the past!

Reply:
Thanks, tip accepted!